catalog/transformer/catalog-transformer-streaming-impl/src/test/resources/multiPolygon.gml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<gml:MultiGeometry xmlns:gml="http://www.opengis.net/gml" srsName="">
```

Comment:
❓ in these GML test files, is the coord ordering lat,lon or lon,lat?  
Shouldn't we have some tests using non-4326 CRSs?